static/css/theme-activestate.css

```diff
@@ -104,4 +183,4 @@ a:hover {
 
 #sidebar hr {
     border-color: var(--MENU-SECTION-HR-color);
-}
+}
```

Comment:
Can you add a line break after this last rule so the file ends with a blank line? github likes that for some reason.